src/components/portfolio/PortfolioBlock.js

```diff
@@ -1,24 +1,49 @@
-import React from 'react';
+import React from "react";
 import IconLink from "./IconLink";
-import {Box} from "@mui/material";
+import { Box } from "@mui/material";
+
+const liveTitles = [
+  "sql query editor view",
+  "results summary component using html, css and flexbox",
+  "e-commerce application using nextjs, prisma, and stripe payments"
+];
```

Comment:
**Suggestion:** Consider using a more efficient way to check if the title is in the liveTitles array. You can use a Set instead of an Array for liveTitles, which will make the lookup operation more efficient.
```suggestion
const liveTitles = new Set([
  "sql query editor view",
  "results summary component using html, css and flexbox",
  "e-commerce application using nextjs, prisma, and stripe payments"
]);
...
{liveTitles.has(title) && (
  <Box p={1} border={"2px solid black"} borderRadius={"25px"}>
    <IconLink link={live} title={"Live Demo"} icon={"fa fa-safari"} />
  </Box>
)}
```